### PR TITLE
request: throw async error for long requests

### PR DESCRIPTION
**Pull request by Zolmeister**

### CODE REVIEW

File: src/lib/request.coffee [hunk 1]
```diff
@@ -32,3 +36,13 @@ module.exports = (url, options) ->
   window.fetch url, options
   .then statusCheck
   .then toJson
+  .then (res) ->
+    endTime = Date.now()
+    requestTime = endTime - startTime
+    if requestTime > MIN_REQUEST_TIME_ERROR_MS
+      setTimeout ->
+        throw new Error "event=long_request
```
Comment (austinhallock) [inline src/lib/request.coffee]: doesn't seem to follow https://github.com/claydotio/wiki/wiki/ops#logs. this uses linebreak instead of comma

Comment (austinhallock) [inline src/lib/request.coffee]: might also be worth tossing navigator.userAgent in here

**Comment by Zolmeister** [comment]
updated

**Comment by Zolmeister** [comment]
bump

**Pull request merged**